benchcab/bench_config.py

```diff
@@ -43,6 +43,32 @@ def check_config(config: dict):
                 "that is compatible with the f90nml python package."
             )
 
+    # the "pbs" key is optional
+    if "pbs" in config:
```

Comment:
I think we will need 2 PBS keys: one for fluxsite and one for spatial as the resources are likely going to be quite different.

So should we rename "pbs" to "pbs_fluxsite" ?

Reply:
Yes I agree. I'm thinking we go with this structure:
```yaml
fluxsite:
  pbs:
    ncpus: 16
    ...
  multiprocessing: True
spatial:
  pbs:
    ncpus: 16
    ...
```
mainly because `multiprocessing` is probably specific to only fluxsite tests. My guess is we won't be running multiple cable MPI executables in parallel on a single PBS job like we do with the fluxsite case (e.g. if we use `payu run` to execute each spatial task, we would only be able to run one task per PBS job). Although it might be possible to run multiple MPI instances on a single PBS job (see [here](https://wiki.bath.ac.uk/display/BalenaHPC/Multiple+MPI+instances+on+a+single+node#MultipleMPIinstancesonasinglenode-Usecase) for example).

Reply:
Yes, that looks good.